demos/using-dev-build/webpack.config.js

```diff
@@ -1,8 +1,6 @@
 const path = require("path");
 const webpack = require("webpack");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
-const TerserPlugin = require("terser-webpack-plugin");
```

Comment:
These were not used, unrelated to the PR.

Reply:
Gotcha! Next time don't hesitate to create another PR